### ISSUE TITLE
Error creating init file locally for GPU + accuracy [spacy-nightly]

### ISSUE THREAD

**Issue by ahalterman** [explosion/spaCy#6340]
Creating an init file locally using the CLI and the GPU produces the following error:

```
python -m spacy init config -p textcat -l en --optimize accuracy  config_trf.cfg
ℹ Generated template specific for your use case
- Language: en
- Pipeline: textcat
- Optimize for: accuracy
- Hardware: GPU
- Transformer: roberta-base
✘ Config validation error
textcat.model.tok2vec -> pooling   field required
```

```
Environment
spaCy version: 3.0.0rc2
Platform: Darwin-19.6.0-x86_64-i386-64bit
Python version: 3.6.11
Pipelines: en_core_web_sm (3.0.0a0), en_core_web_lg (3.0.0a0), en_core_web_trf (3.0.0a0)
```

**Comment by svlandeg** [comment]
Thanks for the report! This is definitely a bug - we're fixing it :-)

**Comment by svlandeg** [comment]
This should hopefully be fixed in PR #6371, but let us know if you run into new issues!
